web/templates/popovers/color_picker_popover.hbs

```diff
@@ -0,0 +1,36 @@
+<div class="popover-menu color-picker-popover" data-simplebar data-simplebar-tab-index="-1">
+    <div class="message_header message_header_stream" data-stream-id="{{stream_id}}">
+        <div class="message-header-contents" style="background: {{recipient_bar_color}};">
+            <div class="message_label_clickable stream_label">
+                <span class="stream-privacy-modified-color-{{stream_id}} stream-privacy filter-icon"  style="color: {{stream_privacy_icon_color}}">
+                    {{> ../stream_privacy .}}
+                </span>
+                {{#if stream_name}}
+                <span class="message-header-stream-name">
+                    {{~stream_name~}}
+                </span>
+                {{else}}
+                &nbsp;
+                {{/if}}
+            </div>
+        </div>
+    </div>
+    <ul role="menu" class="popover-menu-list">
+        <li role="none">
+            <div role="group" class="color-swatch-list" aria-label="{{t 'App theme' }}">
+                {{#each stream_color_palette as |hex_color|}}
+                    <input type="radio" id="color-{{hex_color}}" class="color-swatch-input" name="color-picker-select" data-swatch-color="{{hex_color}}" {{#if (eq hex_color ../stream_color )}}checked{{/if}} />
+                    <label role="menuitemradio" class="color-swatch-label" for="color-{{hex_color}}" style="background-color: {{hex_color}};" aria-label="{{t 'Select automatic theme' }}" tabindex="0"></label>
```

Comment:
This also looks copy-pasted from somewhere.

Reply:
Fixed. Reason is the same as mentioned in https://github.com/zulip/zulip/pull/32063#discussion_r1893234532.